pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -1118,7 +1118,7 @@ func TestEventTransientData(t *testing.T) {
 		svc, err := New(&protocol.MockProvider{})
 		require.NoError(t, err)
 
-		err = svc.AcceptExchangeRequest(generateRandomID())
+		err = svc.AcceptExchangeRequest(generateRandomID(), "", "")
```

Comment:
All test cases here pass empty values. Can we verify non-empty cases as well ?

Reply:
done